apps/vor/pkg/lessonplan/lessonplan.go

```diff
@@ -0,0 +1,76 @@
+package lessonplan
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-playground/validator/v10"
+	"github.com/pkg/errors"
+
+	"github.com/chrsep/vor/pkg/postgres"
+	"github.com/chrsep/vor/pkg/rest"
+)
+
+func NewRouter(server rest.Server, store Store) *chi.Mux {
+	r := chi.NewRouter()
+	r.Method("POST", "/", postNewLessonPlan(server, store))
+
+	return r
+}
+
+func postNewLessonPlan(server rest.Server, store Store) http.Handler {
```

Comment:
Didn't we plan to put POST new plan under the `/class` route/package instead?

Reply:
yeah i forgot about that. i thought there is no class package